jumpscale/packages/chatflows/actors/chatbot.py

```diff
@@ -21,10 +21,10 @@ def new(self, package: str, chat: str, client_ip: str, query_params: dict = None
         chatflow = package.get(chat)
         if not chatflow:
             raise j.exceptions.Value(f"Chat {chat} not found")
-        
+
         obj = chatflow()
         self.sessions[obj.session_id] = obj
-        return {"sessionId": obj.session_id}
+        return {"sessionId": obj.session_id, "title": obj.title}
```

Comment:
can we make sure we always send one incase of obj.title is None ?
e.g `"title": "Zero Chat bot" if obj.title is None else obj.title` ?

Reply:
We have a default title in the base class